webrtc-stats.html

```diff
@@ -1918,6 +1919,17 @@ <h2>
                   <code><a>RTCRemoteInboundRtpStreamStats</a></code> object for the same SSRC.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>rid</code></dfn> of type <span class="idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>
+                  Exposes the <a href="https://w3c.github.io/webrtc-pc/#dom-rtcrtpcodingparameters-rid">rid</a>
+                  encoding parameter of this RTP stream if it has been set, otherwise it is
+                  undefined. If set this value will be present regardless if the RID RTP header
```

Comment:
The value will be present even if the RID RTP header extension has not been negotiated.